CHANGELOG.md

```diff
@@ -52,6 +52,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   previously.
 - Now doesn't force ASCII characters in the NER task for generative models, making the
   target JSON dictionary more consistent with the input text.
+- Samples with excessively short or long texts have been removed, being contexts whose
+  lengths are in the top-5% or bottom-5%.
```

Comment:
While I understand that this is easy to reason about in a text I would much rather have a clearer minimum word boundary and a maximum word boundary. Like 10-3 000 tokens?

Reply:
Sure, we can set them as explicit numbers instead. We should probably do it in terms of number of characters, like what is done here. I had a look at the resulting boundaries from the quantiles, and the upper bound on the QA context is around 3000 characters, and around 80 characters for the question. We could conservatively set it to 5000 and 100 characters, respectively? 

To give some examples of outliers: The ScandiQA datasets have samples that are >100,000 characters, and the SQuAD-nl dataset has questions that are >1000 characters (and which are not questions at all).